packages/miniflare/src/shared/dev-registry.ts

```diff
@@ -18,16 +18,14 @@ import { getGlobalWranglerConfigPath } from "./wrangler";
 
 export type WorkerRegistry = Record<string, WorkerDefinition>;
 
-export type WorkerEntrypointsDefinition = Record<
```

Comment:
is this not used anywhere else?

Reply:
This type was copied from the wrangler dev registry implementation but never get used by any logic here except going through the `WorkerDefinition` type below. I just inlined it into `WorkerDefinition` now as the build script complains `WorkerEntrypointsDefinition` have to be exported all together in the package entrypoint file.